cloud/organization/organization_token.go

```diff
@@ -0,0 +1,190 @@
+package organization
+
+import (
+	httpContext "context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+
+	astrocore "github.com/astronomer/astro-cli/astro-client-core"
+	"github.com/astronomer/astro-cli/cloud/user"
+	"github.com/astronomer/astro-cli/context"
+	"github.com/astronomer/astro-cli/pkg/input"
+	"github.com/astronomer/astro-cli/pkg/printutil"
+)
+
+var (
+	errInvalidOrganizationTokenKey = errors.New("invalid organization token selection")
+	errOrganizationTokenNotFound   = errors.New("organization token specified was not found")
+	errOrgTokenInWorkspace         = errors.New("this organization token has already been added to the workspace")
+	errBothNameAndID               = errors.New("both a token name and id were specified. Specify either the name or the id not both")
+)
+
+const (
+	workspaceConst = "WORKSPACE"
```

Comment:
nit: you could just say `workspaceEntity` and not have a suffix `Const`, since it is used when checking the entity of the token